pkg/cdi/cache.go

```diff
@@ -147,12 +146,12 @@ func (c *Cache) refresh() error {
 		devices    = map[string]*Device{}
 		conflicts  = map[string]struct{}{}
 		specErrors = map[string][]error{}
-		result     []error
+		errs       = []error{}
 	)
 
 	// collect errors per spec file path and once globally
 	collectError := func(err error, paths ...string) {
-		result = append(result, err)
+		errs = append(errs, err)
```

Comment:
```suggestion
		errs = errors.Join(errs, err)
```

Reply:
Yeah, this wasn't producing a natural collection/topology of joint errors. Updated.